src/filters/time.js

```diff
@@ -68,16 +79,41 @@ const formatDate = (date, format) => {
   return formats[format];
 };
 
+/* @docs
+label: getDate
+category: formatting
+note: |
+  Returns a date in any format.
+example: |
+  {{ pubdate | getDate('iso') }}
+params:
+  date:
+    type: date | string
+    note: If no date is given, returns the current date
+  format:
+    type: string
+    note: See js file for a list of available date formats
+*/
 const getDate = (date, format) => {
   date = typeof date === 'string' ? new Date(date) : date || now;
   return format ? formatDate(date, format) : date;
 };
 
+/* @docs
+label: rssDate
+category: rss
```

Comment:
Should this read as `RSS` on the page? It must be going through a capitalize filter instead since it outputs to `Rss`